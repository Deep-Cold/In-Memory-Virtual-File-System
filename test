newDisk 114514
newDir 1919810
changeDir 1919810
newDoc 19260817 txt 8964
changeDir ..
