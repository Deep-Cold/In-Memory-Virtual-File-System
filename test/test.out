newDisk 114514
newDoc nmsl java fku
newDir name
changeDir name
newSimpleCri ab size >= 2
newSimpleCri ac type equals "txt"
newBinaryCri ba ab && ac
